src/sdk.rs

```diff
@@ -372,6 +382,47 @@ pub fn keccak(input: &[u8]) -> H256 {
     }
 }
 
+/// Calls environment ripemd160 on given input.
+pub fn ripemd160(input: &[u8]) -> [u8; 20] {
+    unsafe {
+        const REGISTER_ID: u64 = 1;
+        exports::ripemd160(input.len() as u64, input.as_ptr() as u64, REGISTER_ID);
+        let bytes = [0u8; 20];
+        exports::read_register(REGISTER_ID, bytes.as_ptr() as u64);
+        bytes
+    }
+}
+
+/// Recover address from message hash and signature.
+pub fn ecrecover(hash: H256, signature: &[u8]) -> Result<Address, ECRecoverErr> {
+    unsafe {
+        let hash_ptr = hash.as_ptr() as u64;
+        let sig_ptr = signature.as_ptr() as u64;
+        let recover_register_id = 1;
+        let keccak_register_id = 2;
```

Comment:
Should be consts.

Reply:
Done in https://github.com/aurora-is-near/aurora-engine/pull/190/commits/ede8a4df16419d61b2d50f301b2eae188324c1d3